client/src/api/v2/sensors/status.ts

```diff
@@ -0,0 +1,65 @@
+import { useData } from './data';
+
+export interface SensorStatus {
+  /** Sensor name */
+  name: string;
+  /** Human readable sensor name */
+  label: string;
+  /** Whether sensor is on/off */
+  state: boolean;
+}
+
+interface SensorStates {
+  gps: boolean;
+  power: boolean;
+  cadence: boolean;
+  reed: boolean;
+  accelerometer: boolean;
+  gyroscope: boolean;
+  potentiometer: boolean;
+  thermometer?: boolean;
```

Comment:
Why is `thermometer?: boolean` an optional parameter, while all others are not?

Reply:
No reason, whoops